python/ray/__init__.py

```diff
@@ -24,6 +26,23 @@
     from ctypes import CDLL
     CDLL(so_path, ctypes.RTLD_GLOBAL)
 
+if (platform.system().lower() == "Linux".lower()
+    and "Microsoft".lower() in platform.release().lower()):
+    # WSL's /proc/meminfo has a bug that crashes Python; work around it
```

Comment:
Which line causes the crash?

Reply:
The [`fields = line.split()` line in `virtual_memory()`](https://github.com/giampaolo/psutil/blob/19f87b14f83669fd450e032e35d5b328b9acc118/psutil/_pslinux.py#L393). It splits on spaces and there's a missing space when this behavior is triggered.

Reply:
Or rather, I mean the following line, where it tries to convert `b"kB"` to integer.

Reply:
Can you add a comment that either describes the bug or points to a link describing it?

Reply:
Also a high-level description of the workaround is probably worthwhile.

Reply:
I actually had these in the commit message, but I'll add it in the comment.